sdk/servicebus/Azure.Messaging.ServiceBus/src/Management/SubscriptionDescription.cs

```diff
@@ -324,7 +311,7 @@ public bool Equals(SubscriptionDescription other)
                 return true;
             }
 
-            if (ReferenceEquals(left, null) || ReferenceEquals(right, null))
+            if (left is null || right is null)
```

Comment:
FxCop?

Reply:
Nope, it was this suggestion - 

![image](https://user-images.githubusercontent.com/45216704/83826430-bc20f780-a690-11ea-89af-c9313991ba97.png)